apps/okreads/browser/src/app/app.component.html

```diff
@@ -7,6 +7,7 @@ <h1>okreads</h1>
           <button
             data-testing="toggle-reading-list"
             mat-button
+            aria-label="Toggle Reading List"
```

Comment:
Are you sure this needs an `aria-label`? It has text content that describes what it does. Isn't that sufficient?